nemo_text_processing/text_normalization/ja/data/date/day.tsv

```diff
@@ -0,0 +1,40 @@
+1	一
+01	一
```

Comment:
See my comment in `month.tsv`. Again, if it's taking forever to update, I'd just leave it as is.